README.md

```diff
@@ -1 +1,23 @@
-# trifon
\ No newline at end of file
+# trifon // Loads the latest XKCD comic from
+// xkcd.com and shows it. Works well
+// when run from a Siri Shortcut.
+// The comic will be shown inline and
+// Siri will read out the text that 
+// supplements the comic.
+let url = "https://xkcd.com/info.0.json"
```

Comment:
// Variables used by Scriptable.
// These must be at the very top of the file. Do not edit.
// icon-color: purple; icon-glyph: magic; share-sheet-inputs: url, file-url, plain-text, image;
https://iforgot.apple.com/password/verify/appleid?language=en_GB